src/js/utils/parser.js

```diff
@@ -81,22 +81,26 @@ define([
     /**
      * String representations of booleans and numbers that are 5 characters in length or less
      * are returned typed
+     * Replaced ' with %27 for urls to work
      */
     parser.serialize = function (val) {
         if (val === undefined) {
             return null;
         }
-        if (typeof val === 'string' && val.length < 6) {
-            var lowercaseVal = val.toLowerCase();
-            if (lowercaseVal === 'true') {
-                return true;
-            }
-            if (lowercaseVal === 'false') {
-                return false;
-            }
-            if (!isNaN(Number(val)) && !isNaN(parseFloat(val))) {
-                return Number(val);
+        if (typeof val === 'string') {
+            if (val.length < 6) {
+                var lowercaseVal = val.toLowerCase();
+                if (lowercaseVal === 'true') {
+                    return true;
+                }
+                if (lowercaseVal === 'false') {
+                    return false;
+                }
+                if (!isNaN(Number(val)) && !isNaN(parseFloat(val))) {
+                    return Number(val);
+                }
             }
+            val = val.replace('\'', '%27');
```

Comment:
We shouldn't change this function. Not all values passed to config should be transformed this way. The issue is when we [construct an image url in css.js](https://github.com/jwplayer/jwplayer/blob/master/src/js/utils/css.js#L112). At that point we should `escape()` the value to handle apostrophes and quotes and wrap it in quotes to protect agains parenthesis in the value.